src/App.tsx

```diff
@@ -1,8 +1,20 @@
+import { Navigate, Route, Routes } from 'react-router-dom';
 import './App.scss';
+import { Header } from './components/Header/Header';
+import { HomePage } from './components/HomePage/HomePage';
+import { NotFoundPage } from './components/NotFoundPage/NotFoundPage';
+import { PeoplePage } from './components/PeoplePage/PeoplePage';
 
 const App = () => (
-  <div className="App">
-    <h1>People table</h1>
+  <div className="app container is-max-desktop">
+    <h1 className="app__title">People table</h1>
+    <Header />
+    <Routes>
+      <Route path="/" element={(<HomePage />)} />
+      <Route path="/people" element={(<PeoplePage />)} />
+      <Route path="/home" element={<Navigate to="/" />} />
```

Comment:
use `replace` prop for your `navigate` to avoid infinite cycle while pressing `back` button